shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1847,9 +1853,29 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
   return YES;
 }
 
+- (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
+       shouldReceiveEvent:(UIEvent*)event API_AVAILABLE(ios(13.4)) {
+  if (gestureRecognizer == _continuousScrollingPanGestureRecognizer &&
+      event.type == UIEventTypeScroll) {
+    flutter::PointerData pointer_data = [self generatePointerDataAtLastMouseLocation];
+    pointer_data.device = reinterpret_cast<int64_t>(_continuousScrollingPanGestureRecognizer);
+    pointer_data.kind = flutter::PointerData::DeviceKind::kTrackpad;
+    pointer_data.signal_kind = flutter::PointerData::SignalKind::kScrollInertiaCancel;
+
+    if (event.timestamp < _scrollInertiaEventAppKitDeadline) {
```

Comment:
can you comment out that if scroll happens before the "app kit deadline", it's likely that the cancel event is caused by user and not by the system.

Reply:
Will change it :+1: